hassio/src/dialogs/network/dialog-hassio-network.ts

```diff
@@ -119,6 +119,7 @@ export class DialogHassioNetwork
                     html`<mwc-tab
                       .id=${device.interface}
                       .label=${device.interface}
+                      dialogInitialFocus
```

Comment:
How can it be multiple elements with `dialogInitialFocus`? which one "win"?

Reply:
first